src/main/connector/system/Linux/Linux.yaml

```diff
@@ -0,0 +1,156 @@
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+      - "linux"
+    criteria:
+      - type: osCommand
+        commandLine: "/usr/bin/uname -o"
+        expectedResult: "GNU/Linux"
+        errorMessage: "Not a valid Linux host."
+monitors:
+  cpu:
+    simple:
+      type: multiInstance
+      sources:
+        source(1):
+          # cpuId;user;nice;system;idle;iowait
+          type: osCommand
+          commandLine: /usr/bin/cat /proc/stat
+          computes:
+            - type: awk
+              script: /cpu[0-9]/ {sub("cpu",""); print $1 ";" $2 / 1000 ";" $3 / 1000 ";" $4 / 1000 ";" $5 / 1000 ";" $6 / 1000}
+      mapping:
+        source: "${source::monitors.cpu.simple.sources.source(1)}"
+        attributes:
+          id: $1
+          name: ${awk::sprintf("%s %s", "cpu", $1)}
+          system.cpu.logical_number: $1
+        metrics:
+          system.cpu.utilization{system.cpu.state="user"}: rate($2)
+          system.cpu.utilization{system.cpu.state="nice"}: rate($3)
+          system.cpu.utilization{system.cpu.state="system"}: rate($4)
+          system.cpu.utilization{system.cpu.state="idle"}: rate($5)
+          system.cpu.utilization{system.cpu.state="io_wait"}: rate($6)
+          system.cpu.time{system.cpu.state="user"}: $2
+          system.cpu.time{system.cpu.state="nice"}: $3
+          system.cpu.time{system.cpu.state="system"}: $4
+          system.cpu.time{system.cpu.state="idle"}: $5
+          system.cpu.time{system.cpu.state="io_wait"}: $6
+  memory:
+    simple:
+      sources:
+        source(1):
+          # memTotal;memFree;memUsed;memBuffers;memCached;memFreeUtilization;memUsedUtilization;memBuffersUtilization;memCachedUtilization
+          type: osCommand
+          commandLine: cat /proc/meminfo
+          computes:
+            - type: awk
+              script: "${file::memory.awk}"
+      mapping:
+        source: "${source::monitors.memory.simple.sources.source(1)}"
+        attributes:
+          id: memory_usage
+          system.memory.limit: $1
+        metrics:
+          system.memory.usage{system.memory.usage="free"}: $2
+          system.memory.usage{system.memory.usage="used"}: $3
+          system.memory.usage{system.memory.usage="buffers"}: $4
+          system.memory.usage{system.memory.usage="cached"}: $5
+          system.memory.utilization{system.memory.usage="free"}: $6
+          system.memory.utilization{system.memory.usage="used"}: $7
+          system.memory.utilization{system.memory.usage="buffers"}: $8
+          system.memory.utilization{system.memory.usage="cached"}: $9
+  disk:
+    simple:
+      type: multiInstance
+      sources:
+        source(1):
+          # id;reads;readsMerged;readTime;writes;writesMerged;writeTime;ioTime
+          type: osCommand
+          commandLine: cat /proc/diskstats
+          computes:
+            - type: awk
+              script: '{print $3 ";" $4 ";" $5 ";" $7 / 1000 ";" $8 ";" $9 ";" $11 / 1000 ";" $13 / 1000}'

```

Comment:
It's probably more efficient to use 3 `divideBy` computes, rather than an AWK script to divide by 1000.